src/neptune/exceptions.py

```diff
@@ -790,26 +790,27 @@ def __init__(self, project: QualifiedName):
         super().__init__(message.format(project=project, **STYLES))
 
 
-class NeptuneIntegrationNotInstalledException(NeptuneException):
-    def __init__(self, integration_package_name, framework_name):
+class NeptuneMissingRequirementException(NeptuneException):
+    def __init__(self, package_name: str, framework_name: Optional[str]):
         message = """
-{h1}
-----NeptuneIntegrationNotInstalledException-----------------------------------------
-{end}
-Looks like integration {integration_package_name} wasn't installed.
-To install, run:
-    {bash}pip install {integration_package_name}{end}
-Or:
-    {bash}pip install "neptune[{framework_name}]"{end}
-
-You may also want to check the following docs page:
-    - https://docs.neptune.ai/integrations
-
-{correct}Need help?{end}-> https://docs.neptune.ai/getting_help
-"""
+    {h1}
+    ----NeptuneMissingRequirementException-----------------------------------------
+    {end}
+    Looks like package {package_name} wasn't installed.
+    To install, run:
+        {bash}pip install {package_name}{end}
+    Or:
+        {bash}pip install "neptune[{framework_name}]"{end}
+
+    You may also want to check the following docs page:
```

Comment:
Maybe this makes things slightly more clear and specific?

```suggestion
    Looks like the {package_name} package isn't installed.
    To install it, run:
        {bash}pip install {package_name}{end}
    Or install both Neptune and the integration:
        {bash}pip install "neptune[{framework_name}]"{end}

    For detailed instructions, check the integration guides:
```